code/modules/research/xenobiology/crossbreeding/reproductive.dm

```diff
@@ -18,15 +18,6 @@ Reproductive extracts:
 	if((last_produce + cooldown) > world.time)
 		to_chat(user, span_warning("[src] is still digesting!"))
 		return
-	if(istype(O, /obj/item/storage/bag/bio))
-		var/list/inserted = list()
-		SEND_SIGNAL(O, COMSIG_TRY_STORAGE_TAKE_TYPE, /obj/item/reagent_containers/food/snacks/monkeycube, src, 1, null, null, user, inserted)
-		if(inserted.len)
-			var/obj/item/reagent_containers/food/snacks/monkeycube/M = inserted[1]
```

Comment:
is it perhaps due to this line that it assumes the top item is a monkey cube? have you tried it with only monkey cubes in the bag?

Reply:
i could have sworn it's worked for me in the past, so that might be the issue